README.md

```diff
@@ -12,7 +12,9 @@ In the European Union, this extension's fields correspond to [eForms BG-611 (Con
 
 The `financeType.csv` codelist is based on the list on [Page 57 of the World Bank PPP Disclosure Framework](http://pubdocs.worldbank.org/en/143671469558797229/FrameworkPPPDisclosure-071416.pdf#page=57)
 
-## Example
+## Examples
```

Comment:
I suggest changing the order of examples, since contracting process financing is much more common that individual contract financing.